### PR TITLE
Update TPC reco settings for data and MC

### PR DESCRIPTION
**Pull request by chiarazampolli**
@wiechula @sawenzel 
Needs to be validated by @wiechula

### CODE REVIEW

File: MC/run/ANCHOR/2021/OCT/pass4/anchorMC.sh [hunk 1]
```diff
@@ -27,6 +27,10 @@ RUNNUMBER=${ALIEN_JDL_LPMRUNNUMBER:-505673}
 # the script location can be configured with a JDL option
 cp ${ALIEN_JDL_ASYNCRECOSCRIPT:-$O2DPG_ROOT/DATA/production/configurations/2021/OCT/apass4/async_pass.sh} async_pass.sh
 cp $O2DPG_ROOT/DATA/production/configurations/2021/OCT/${ALIEN_JDL_LPMPASSNAME:-apass4}/setenv_extra.sh .
+#settings that are MC-specific
+sed -i 's/--dEdxUseFullGainMap --dEdxDisableResidualGainMap /--dEdxDisableTopologyPol --dEdxDisableGainMap --dEdxDisableResidualGainMap --dEdxDisableResidualGain/' setenv_extra.sh
+sed -i 's/export CONFIG_EXTRA_PROCESS_o2_gpu_reco_workflow="/&GPU_global.dEdxSplineTopologyCorrFile=splines_for_dedx_threshold_3.5.root;/' setenv_extra.sh
```
Comment (sawenzel) [inline MC/run/ANCHOR/2021/OCT/pass4/anchorMC.sh:32]: so far such DPL arguments (everything non-ConfigurableParam) will not be automatically picked up. Some more work (or better configurability) is necessary to transfer this to MC.

Comment (chiarazampolli) [reply MC/run/ANCHOR/2021/OCT/pass4/anchorMC.sh:31]: Hi @sawenzel , 

I removed the options, but the config key values should be fine, right?

**Comment by chiarazampolli** [comment]
@wiechula, @sawenzel , what about now? For data we need surely to change.

**Review by wiechula — APPROVED** [review]
looks fine for me. For the apass4 anchored MC we'll then need to update the settings again slightly.

**Review by sawenzel — APPROVED** [review]

**Comment by sawenzel** [comment]
Looks good to me and should be picked up automatically by MC.

**Comment by sawenzel** [comment]
@wiechula : Can we make --TPCuseCCDB a default option for TPC digitization? In which cases would this be a problem?

**Pull request merged**